about.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Austin IWW - About</title>
+</head>
+<body>
+    <div>
+        <h1>About Austin IWW</h1>
+        <h2>Mission Statement</h2>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt ducimus quos ex pariatur dolore vitae atque aspernatur. Cum libero doloribus, iste eos ab ratione ea vero harum sequi maxime facere!</p>
+        <h2>Frequently Asked Questions</h2>
+            <ul>
+                <li>
+                    <p>Question #1</p>
+                    <p>Answer #1</p>
+                </li>
+                <li>
+                    <p>Question #2</p>
+                    <p>Answer #2</p>
+                </li>
+                <li>
+                    <p>Question #3</p>
+                    <p>Answer #3</p>
+                </li>
+            </ul>
+        <h2>Contact</h2>
+            <ul>
+                <li><a href=""> Austin IWW on Discord</a></li>
+                <li>[insert email address here]</li>
```

Comment:
We'll want this to be an @iwwaustin.com/@iwwaustin.org address ideally - will investigate options.